enhancement: add `forc index revert` command (#609)
FuelLabs/fuel-indexer: docs/src/reference-guide/plugins/forc-index/index.md

    init      Create a new indexer project in the current directory
    new       Create a new indexer project in a new directory
    remove    Stop and remove a running index

>     revert    Revert a running index to the last deployed version

Reviewer: I should probably clarify some terminology at our next team sync as I think I started this trend of calling indexers indices 😅 
```suggestion
    revert    Revert a running indexer to its last previous version
```
